par_system:
        PAR_SYSTEM_SAMPLE_RATE: 10.0
